src/main/java/bookstore/model/CartItem.java

```diff
@@ -0,0 +1,41 @@
+package bookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+
+@Getter
+@Setter
+@Entity
+@SQLDelete(sql = "UPDATE books SET is_deleted = true WHERE id=?")
```

Comment:
```suggestion
@SQLDelete(sql = "UPDATE cart_items SET is_deleted = true WHERE id=?")
```
be attentive with copy-paste ;)